package.json

```diff
@@ -33,6 +33,7 @@
     "yarn": ">=1.19.0 < 2.0.0"
   },
   "dependencies": {
+    "@boxyhq/saml-jackson": "0.2.2",
```

Comment:
We should discuss if we would want to upgrade our base node version in `.nvmrc`, this will affect all of our self-hosted users. This module is expecting a higher version:

`error @boxyhq/saml-jackson@0.2.2: The engine "node" is incompatible with this module. Expected version ">=14.18.1". Got "14.17.6"`

Reply:
@baileypumfleet @PeerRich

Reply:
I can set it back, not a problem